### PR TITLE
implement-loginPage

### PR DESCRIPTION
**Pull request by yosrada**

### CODE REVIEW

**Comment by Karyum** [comment]
@tawfiknasser @majdya For the sake of urgency I am merging this immediately, and working a long students to get the critical error sorted out, it should have sorted out on the same day that it happened.

@yosrada I have reviewed the pull request, thanks for the contribution, I didn't quite understand the reason behind the second login file, but I will keep it in the pull request for future reference if you need (but I will remove it from the dev for now)

**Review by Karyum — APPROVED** [review]

**Comment by Karyum** [comment]
# AGAIN MERGING THIS FOR QUICK FIX

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -5,9 +5,36 @@ import { Password } from 'primereact/password'
...
+  const RESPONSE_STATUS = {
+    FAIL: false,
+    SUCSSESS: true
+  }
+  const submitUser = async () => {
+    try {
+      if (Email.length === 0 || password.length === 0) setError(true)
+      else {
+        const data = { Email: Email, password: password }
+        const response = await fetch('http://localhost:4000/login', {
```
Comment (tawfiknasser) [inline src/pages/Login/Login.jsx:21]: @yosrada can you refactor this to use `config.js` ?

File: src/pages/login/login.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(data)
+    })
+    const json_response = await response.json()
+    console.log(response.body)
+  } catch (error) {
+    console.log(error)
+  }
+}
```
Comment (tawfiknasser) [inline src/pages/login/login.jsx:85]: @yosrada is tgis duplicated code?
There is `Login.jsx` and there is `login.jsx`

**Pull request merged**